src/pages/report/assessment-report/StorageOverview.tsx

```
@@ -1,61 +1,176 @@
   ContentVariants,
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  Flex,
+  FlexItem,
+  MenuToggle,
+  MenuToggleElement,
 } from '@patternfly/react-core';
 
 import MigrationChart from '../../../components/MigrationChart';
+import { DiskSizeTierSummary } from '@migration-planner-ui/api-client/models';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)